lib/discourse_chat_integration/provider/powerautomate/powerautomate_provider.rb

```diff
@@ -0,0 +1,139 @@
+# frozen_string_literal: true
+
+module DiscourseChatIntegration::Provider::PowerAutomateProvider
+  PROVIDER_NAME = "powerautomate".freeze
```

Comment:
We have `#frozen_string_literal: true` on the file, so no need for this
```suggestion
  PROVIDER_NAME = "powerautomate"
```

(I realise it was probably copied from another provider, which was written before we enabled frozen_string_literal everywhere 👌 )